test/new-e2e/tests/containers/ecs_test.go

```diff
@@ -61,6 +61,7 @@ func (suite *ecsSuite) SetupSuite() {
 		"ddagent:deploy":                             auto.ConfigValue{Value: "true"},
 		"ddagent:fakeintake":                         auto.ConfigValue{Value: "true"},
 		"ddtestworkload:deploy":                      auto.ConfigValue{Value: "true"},
+		"ddagent:dualshipping":                       auto.ConfigValue{Value: "true"},
```

Comment:
I think we should only pass the param for `eks` and `kind` test. Because no option allows us to configure dual shipping from the agent on ECS and docker environment.
cf: https://github.com/DataDog/test-infra-definitions/pull/1283/files#r1878184208